src/Components/Admins/AdminForm/AdminForm.jsx

```diff
@@ -4,19 +4,25 @@ import Modal from '../../Shared/Modal';
 import { useHistory, useParams } from 'react-router-dom';
 import styles from './adminForm.module.css';
 import TextInput from '../../Shared/TextInput/index';
+import { useSelector, useDispatch } from 'react-redux';
+import { postAdmin, putAdmin } from '../../../redux/admins/thunks';
+import { POST_ADMIN_SUCCESS, PUT_ADMIN_SUCCESS } from '../../../redux/admins/constants';
+import Spinner from '../../Shared/Spinner/spinner';
 
 const AdminForm = () => {
   const { id } = useParams();
   const history = useHistory();
-  const [name, setName] = useState();
-  const [lastName, setLastName] = useState();
-  const [email, setEmail] = useState();
-  const [password, setPassword] = useState();
-  const [adminId, setAdminId] = useState();
+  const [name, setName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
   const [edit, setEdit] = useState(false);
   const [showModal, setShowModal] = useState(false);
   const [isActionModal, setIsActionModal] = useState(false);
-  const [serverError, setServerError] = useState();
+  const admins = useSelector((state) => state.admins.list);
```

Comment:
I think that we could have only one selector:
```suggestion
  const { list: admins, isLoading, error } = useSelector((state) => state.admins);
```

Reply:
done!